src/applications/mhv-secure-messaging/tests/e2e/keyboard-nav-test/secure-messaging-keyboard-nav-to-links-buttons-landing-page.cypress.spec.js

```diff
@@ -25,14 +25,14 @@ describe('Secure Messaging Verify Links and Buttons Keyboard Nav', () => {
       cy.get(el).should('have.focus');
     });
 
-    cy.get(Locators.ALERTS.BACK_TOP).scrollIntoView();
+    // cy.get(Locators.ALERTS.BACK_TOP).scrollIntoView();
 
-    cy.get(Locators.ALERTS.BACK_TOP)
-      .shadow()
-      .find('.docked.reveal', { timeout: 1000 })
-      .should('be.visible');
-    cy.realPress('Tab');
-    cy.get(Locators.ALERTS.BACK_TOP).should('have.focus');
+    // cy.get(Locators.ALERTS.BACK_TOP)
+    //   .shadow()
+    //   .find('.docked.reveal', { timeout: 1000 })
+    //   .should('be.visible');
+    // cy.realPress('Tab');
+    // cy.get(Locators.ALERTS.BACK_TOP).should('have.focus');
```

Comment:
QUESTION: Why was this commented out?

Reply:
@mdewey, I believe this was Alex that approved the skip of this test.

Reply:
Sounds good. Thanks @rmessina1010 . 

@oleksii-morgun is this work captured somewhere?

Reply:
@mdewey yes, it's on our radar. @fazilqa has a pending fix that is being tested